schema/README.md

```diff
@@ -724,8 +687,9 @@ An example of a `Copyright` and a derived `Right`:
         "http://schema.org/",
         "<coalaip placeholder>"
     ],
-    "@type": "<coalaip placeholder>/Copyright",
+    "@type": "<coalaip placeholder>/Right",
     "@id": "<URI pointing to this object>",
+    "rightType": "Copyright",
     "rightsOf": "<URI pointing to a CreativeWork object (usually should be a Manifestation)>",
```

Comment:
this generally makes good sense to me, although I'm still a little bit confused because in some instances `rightsOf` is said to point to a `CreativeWork` and in others it still says that it points to a URI of a `Copyright` object. Wouldn't Copyright objects no longer exist if that schema is removed from the spec?

Reply:
I think an example of what what a `rightsOf` property could point to would help clarify here. For example in the case of Ujo we are issuing `Right` objects that reference `MusicRecording` objects. Would `rightOf` point to the MusicRecording object in JSON format, or a legal document that is hashes and stored on chain? At the moment in our implementation we are not populating this field due to this uncertainty. We are hoping to migrate the data to be inline with the spec though.

Reply:
Another potentially related question here is how Rights are expected to be referenced from a `CreativeWork`. In other words, if I have a `CreativeWork` object, should it contain an array of `Right` objects? In our case this is how we handle it in order to reference `percentageShares` on a `MusicRecording` and then split payments that are made on chain. This seems like a common usecase for these `Right` objects, but the way we are handling it currently seems potentially out of line with this spec.

Reply:
> Wouldn't Copyright objects no longer exist if that schema is removed from the spec?

In this PR I'm proposing to not completely remove them but to make Copyright part of the Right entity by introducing a `rightType: Copyright`, like proposed in the RRM.

> Would `rightOf` point to the MusicRecording object in JSON format, or a legal document that is hashes and stored on chain?

`rightOf` would always point to the MusicRecording object in JSON format. We have the property `license` in `Right` that points to the legal document.

> Another potentially related question here is how Rights are expected to be referenced from a `CreativeWork`.
> In other words, if I have a `CreativeWork` object, should it contain an array of `Right` objects?

@vrde and I stumped upon the same issue when we did some work for the OMI: https://github.com/COALAIP/specs/issues/24 I'm afraid we currently don't have a good answer in the specification for this problem. Additionally, it's specific to the blockchain or DLT you're saving the Right onto. IPFS e.g. is not searchable, so backward pointing links are required. Other solutions might one day be available where searching is possible. Until then, we need a makeshift solution. @alexanderattar would you mind adding this use case somewhere in this repo as an issue?

Reply:
Aha..okay yeah that makes sense. I'm happy to open a separate issue for this